server/routes/api/conversations.js

```diff
@@ -3,6 +3,18 @@ const { User, Conversation, Message } = require("../../db/models");
 const { Op } = require("sequelize");
 const onlineUsers = require("../../onlineUsers");
 
+const getUnreadMessages = (convo) => {
+  let unreadMessages = 0
+  for (let i = convo.messages.length - 1; i > 0; i--) {
+    let message = convo.messages[i]
+    if (message.unread && message.senderId === convo.otherUser.id) {
+      unreadMessages++
+      continue
+    }
+    return unreadMessages
```

Comment:
```suggestion
    if (!message.unread) return unreadMessages;
    if (message.senderId === convo.otherUser.id) unreadMessages++;
```
Personal opinion: this is slightly clearer way of writing the same logic.